src/main/java/edu/harvard/h2ms/service/EventServiceImpl.java

```diff
@@ -38,10 +38,10 @@ public void setEventRepository(EventRepository eventRepository) {
     public Map<String, Long> findEventCountByTimeframe(String timeframe) {
 
         List<Event> events = Lists.newArrayList(eventRepository.findAll());
-        log.info("No. of events found: " + events.size());
+        log.info("No. of events found: ", events.size());
```

Comment:
Does this work? I'm pretty sure this needs to be
`log.info("No. of events found: {0}", events.size());`
instead.